xbmc/settings/windows/GUIWindowSettingsScreenCalibration.cpp

```diff
@@ -178,15 +203,18 @@ bool CGUIWindowSettingsScreenCalibration::OnMessage(CGUIMessage& message)
     break;
   case GUI_MSG_CLICKED:
     {
-      // clicked - change the control...
+      // On click event select the next control
       NextControl();
     }
     break;
   case GUI_MSG_NOTIFY_ALL:
     {
-      if (message.GetParam1() == GUI_MSG_WINDOW_RESIZE)
+      if (message.GetParam1() == GUI_MSG_WINDOW_RESIZE && IsActive())
       {
+        m_Res.clear();
```

Comment:
maybe group this into some private method just like you have above for `NextControl()`